test/unit/mock/cmd/bash-c-type-nginx

```diff
@@ -0,0 +1 @@
+nginx is /usr/sbin/nginx
```

Comment:
This reduces the need for those `type` command mocks, and lets you write test for both when the command exists and when it doesn't.